server/BraveAlerterConfigurator.js

```diff
@@ -45,6 +46,23 @@ class BraveAlerterConfigurator {
     return alertSession
   }
 
+  async createAlertSessionFromSession(session) {
```

Comment:
Need to write unit tests for this.

Reply:
Did not end up writing unit tests. Instead, this code is covered by the integration tests in server/test/integration/BraveAlerterConfiguratorTest/getAlertSessionBySessionidAndAlertApiKeyTest.js and server/test/integration/BraveAlerterConfiguratorTest/getAlertSessionByPhoneNumberTest.js